homework.md

```diff
@@ -4,110 +4,112 @@
 
 What is 2 + 2?
 
-5
+Linus, 2+2 is 4, not 5, PUT ATTENTION STOP PLAYING POKEMON GO ON CLASSES!
 
 ## Question 2
 
 What is the opposite of Black?
 
-Purple
+White, Linus are you daltonic?
 
 ## Question 3
 
 What is the  2 * 3 * 6 * 1 * 10 * 34?
 
-1322
+12240, I need to accept I didn't even knew this.
 
 ## Question 4 
 
 What does HTML stand for?
 
-Hyper Tall Massive Language
+Hypertext Markup Language (HTML) (I didn't even knew this)
 
 ## Question 5
 
 What is CSS useful for?
 
-Making a website go faster
+Making a website looks pretty, the hype for graphic designers and editors.
 
 ## Question 6
 
 Using CSS, how would I select every `<h1>` tag on the page?
 
 ```css
-#h1 {
+<h1> { Here is the section for heading elements
```

Comment:
Hi Rafael,
nice touch with the answers so far :),
I think you are using html tags in css, not sure if that is correct, should be: `h1 {
//css code here
}`
what do you think?

Reply:
Thank you, I felt really stocked on it. Great feedback!